src/libraries/System.Private.CoreLib/src/System/Numerics/IBinaryInteger.cs

```diff
@@ -43,20 +43,63 @@ public interface IBinaryInteger<TSelf>
         /// <returns>The number of trailing zeros in <paramref name="value" />.</returns>
         static abstract TSelf TrailingZeroCount(TSelf value);
 
-        /// <summary>Gets the length, in bits, of the shortest two's complement representation of the current value.</summary>
-        /// <returns>The length, in bits, of the shortest two's complement representation of the current value.</returns>
-        long GetShortestBitLength();
-
         /// <summary>Gets the number of bytes that will be written as part of <see cref="TryWriteLittleEndian(Span{byte}, out int)" />.</summary>
         /// <returns>The number of bytes that will be written as part of <see cref="TryWriteLittleEndian(Span{byte}, out int)" />.</returns>
         int GetByteCount();
 
+        /// <summary>Gets the length, in bits, of the shortest two's complement representation of the current value.</summary>
+        /// <returns>The length, in bits, of the shortest two's complement representation of the current value.</returns>
+        int GetShortestBitLength();
+
+        /// <summary>Tries to write the current value, in big-endian format, to a given span.</summary>
+        /// <param name="destination">The span to which the current value should be written.</param>
+        /// <param name="bytesWritten">The number of bytes written to <paramref name="destination" />.</param>
+        /// <returns><c>true</c> if the value was succesfully written to <paramref name="destination" />; otherwise, <c>false</c>.</returns>
```

Comment:
By the by; my understanding is that `<c>true</c>` and `<c>false</c>` should actually be `<see langword="true" />` and `<see langword="false"/>` so that they capitalize when changing to the VB version. (Same with `null` => Nothing)

Carlos' tool probably handles those as special-cased; but thought I'd share for future reference.

Reply:
@carlossanlop is this handled or should I be explicitly using `see langword` ?